src/test_request.erl

```diff
@@ -51,6 +51,9 @@ delete(Url) ->
 delete(Url, Opts) ->
     request(delete, Url, [], [], Opts).
```

Comment:
@eiri: I used the same approach as in other functions get/post/put. We just call request from there. Personally I prefer the approach you suggest. Should I refactor all of them for consistency?